package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "user",
+  "version": "1.0.0",
+  "description": "This module handles the user operations, assign users to roles and add permissions to roles.",
+  "main": " ",
+  "dependencies": {
+    "commander": "^7.2.0",
```

Comment:
I'm not sure about these deps at all. Are these peer dependencies for auto-changelog?